plenum/server/validator_info_tool.py

```diff
@@ -432,7 +432,8 @@ def __replicas_status(self):
             replica_stat["Watermarks"] = "{}:{}".format(replica.h, replica.H)
             replica_stat["Last_ordered_3PC"] = self._prepare_for_json(replica.last_ordered_3pc)
             stashed_txns = {}
-            stashed_txns["Stashed_checkpoints"] = self._prepare_for_json(len(replica._checkpointer._stashed_recvd_checkpoints))
+            # TODO: Rename? Remove?
```

Comment:
We should not change the output of "validator_info".

Reply:
Generally I agree with "don't change validator_info", however in this case:
- `Stashed_checkpoints` is quite an internal metric, I doubt anyone except probably some developers will ever notice its change or removal
- internal logic changed a lot, so renaming it accordingly may make sense not only from consistency point of view, but also for alerting those who do use this metric that internals did change

Reply:
I agree with your points, but any changes in the validator info outputs need to be checked with Mike B., Dan F. and Lynn since they have some scripts parsing the output JSON, and it may rely on some fields. We've just faced such issues in the past, so it's more safe to check with them before doing any changes there.